### DISCUSSION TITLE
"Say ()" Block in the Stage

### DISCUSSION THREAD

**Discussion by Hamsteroonie** [ScratchAddons/ScratchAddons#7940] [Ideas]
If you drag a "say ()" block (or any of its variants) from a sprite to the stage it will still work, even though it is not available in the block picker. It would be useful if these blocks were available in the block picker directly.
This might already exist and I just haven't been able to find it, if so sorry 😅

**Comment by SiIentNinja** [comment]
I suggested this before #7905
I was going to make a separate discussion just for this, but then I saw this.

Could be useful if you are making an animation in the backdrop, or for debugging
